DataFormats/HeavyIonEvent/interface/HFFilterStruct.h

```diff
@@ -0,0 +1,13 @@
+#ifndef HEAVYIONEVENT_HFFILTERSTRUCT_H
```

Comment:
The convention for new files is to follow the exact path of the header

```C++
#ifndef DataFormats_HeavyIonEvent_HFFilterStruct_H
```

Reply:
implemented in the commit f5397ec4d90e6b54c5f6b056650ebb6cb334036c